### PR TITLE
Add resolve and resolveID

### PR DESCRIPTION
**Pull request by lovinglyy**
**Please describe the changes this PR makes and why it should be merged:**
This small pull request simply create [resolve](https://discord.js.org/#/docs/main/master/class/GuildStore?scrollTo=resolve) and [resolveID](https://discord.js.org/#/docs/main/master/class/GuildStore?scrollTo=resolveID) for GuildStore, respecting the docs/comments in the file.

**Status**
- [x] Code changes have been tested against the Discord API, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/stores/GuildStore.js [hunk 1]
```diff
@@ -27,6 +27,12 @@ class GuildStore extends DataStore {
    * @param {GuildResolvable} guild The guild resolvable to identify
    * @returns {?Guild}
    */
+   resolve(guild) {
+    if (guild instanceof Guild) {
```
Comment (kyranet) [inline src/stores/GuildStore.js]: This should be:

```javascript
if (guild instanceof Guild) return guild;
```

Guild has no property of `guild`.

**Comment by 1Computer1** [comment]
It's inherited from `DataStore`.

**Comment by lovinglyy** [comment]
@1Computer1  oh wow, just tested without the pull request code and it works haha, so cause of another behaviors it didn't work when I tested first, my bad.

**Comment by bdistin** [comment]
While the code would have been wrong, this pr does bring up other items that can be resolved into a guild. `return guild & guild.guild || super.resolve(guild);` would add Role, GuildChannel and Message support for resolving guilds...

**Comment by lovinglyy** [comment]
First I was going to add Message there(that's why the guild.guild around), and maybe the code worked cause I did tests by putting the id and guild object directly instead of getting the Guild object somewhere else in my code(that was returning undefined), then I did the pr. Well was really my bad here, I opened the source after the code not working and didn't notice the inheritance.
Also, that additions with Role, GuildChannel and Message can situationally be in handy.

**Pull request closed without merging**